app/src/main/res/layout/import_wallet_first_layout.xml

```diff
@@ -0,0 +1,162 @@
+          />
+
+      <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/label_input"
+          style="@style/TextInputLayoutAppearanceOutlined"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="28dp"
+          android:layout_marginEnd="16dp"
+          android:maxHeight="191dp"
+          app:layout_constraintTop_toBottomOf="@id/import_title"
+          >
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/keystore_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:gravity="top"
+            android:hint="@string/import_wallet_keystore_private_keys"
+            android:imeOptions="actionUnspecified|flagNoFullscreen|flagNoExtractUi"
+            />
+      </com.google.android.material.textfield.TextInputLayout>
+
+      <View
+          android:id="@+id/first_half_separator"
+          android:layout_width="0dp"
+          android:layout_height="2dp"
+          android:layout_marginEnd="12dp"
+          android:background="#f3f3f3"
+          app:layout_constraintBottom_toBottomOf="@id/or_text"
+          app:layout_constraintEnd_toStartOf="@id/or_text"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="@id/or_text"
+          />
+
+      <View
+          android:id="@+id/second_half_separator"
+          android:layout_width="0dp"
+          android:layout_height="2dp"
+          android:layout_marginStart="12dp"
+          android:background="#f3f3f3"
+          app:layout_constraintBottom_toBottomOf="@id/or_text"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toEndOf="@id/or_text"
+          app:layout_constraintTop_toTopOf="@id/or_text"
+          />
+
+      <TextView
+          android:id="@+id/or_text"
+          style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="8dp"
+          android:text="@string/common_or"
+          android:textColor="#99000000"
+          android:textSize="14sp"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toBottomOf="@id/label_input"
+          />
+
+      <Button
+          android:id="@+id/import_from_file_button"
+          style="@style/ButtonStyle.Light"
+          android:layout_width="match_parent"
+          android:layout_height="@dimen/button_height"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="12dp"
+          android:layout_marginEnd="16dp"
+          app:layout_constraintTop_toBottomOf="@id/or_text"
+          />
+      <LinearLayout
```

Comment:
Check if we can't use a CompoundDrawable here

Reply:
Checked, only way to do this was by using MaterialTheme, and I think that had problems to

Reply:
This will be removed as this development is supposed to be done in a different ticket